shadan-abdulbasit/style.css

```diff
@@ -0,0 +1,139 @@
+body{
+  margin:0px;
+  padding:0px;
+  font-family:"Helvetica";
+}
+.container{
+  display: flex;
+ 
+} 
+
+aside{
+  
+  flex: 2;
+  background: #3d4f5d;
+  display:flex;
+  align-items:flex-start;
+  text-align: right;
+  color: white;
+  padding-left: 5px;
+  margin-right: 10px;
+  padding: 200px 0;
+  
+}
+
+aside div {
+  padding-left: 30px;
+  position:fixed; 
+
+  
+}
+
+aside div p a{
+  border: 1px solid white;
+  padding: 3px 10px;
+  text-decoration:none;
+  color:white;
+  /* font-weight:bold; */
+  margin-right:2px;
+  margin: 0 4px;
+}
+
+main{
+  flex:7;
+  padding: 2.5em;
+}
+#blog{
+  margin-bottom:-12px;
+  font-size:28px;
+  /* font-size: 24px; */
+}
+
+#pined-post , #resent-post{
+  margin-bottom:3em; 
+}
+
+main section .title{
+  color:rgba(128, 128, 128, 0.4);
+  /* border-bottom: 1px solid rgba(128, 128, 128, 0.2); */
+  display: block;
+  font-size:.8em;
+  letter-spacing: .1em;
+  margin-bottom: -18px;
+}
+
+.header{
+  margin: 2em 0;
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: -27px;
+}
+
+.header img{
+  width: 5%;
+  border-radius: 50%;
+}
+
+section a{
+  color: skyblue;
+  text-decoration: none;
+}
+
+p.gray-text{
+  color:#C7C7C7;
+}
+
+.content{
+  font-weight: 100;
+  color: black;
+  line-height: 1.5  ;
+}
+
+.sub{
+  color: white;
+  padding: 3px 17px;
+  border-radius: 4px;
+  margin: 0 3px;
+  
+}
+
+#css{background: #5DB85B;}
+#bg-blue{background: #4C85D5;}
+#bg-red{background: #E5284B;}
+#not-visted{background: #9e9e9e;}
+
+.img-wrapper{
+  display: flex;
+  align-items: flex-start;
+  
+}
+
+.img-wrapper img{
+
+  width: 90%;
+  height: 400px;
+  margin: 30px 0;
+  margin-right: 20px;
+  border: 1px solid gray;
+  border-radius: 2px;
+ 
+}
+
+
+.img1{
+  flex: 1;
+  /* border:2px solid cadetblue; */
+  
+ 
+  
+}
+.img2{
```

Comment:
the classes `.img1` and `.img2` has exactly the same content which is `flex: 1;` so you can remove one of those classes and use the other class for both images